ui/component/or-asset-viewer/src/index.ts

```diff
@@ -1105,6 +1124,9 @@ export class OrAssetViewer extends subscribe(manager)(translate(i18next)(LitElem
     @property({type: String})
     public assetId?: string;
 
+    @property({type: Array})
+    public assetsIds: string[] = [];
+
```

Comment:
Why do we need two properties for selected assets?

Reply:
Asset viewer is only containing info for one asset (to show it), here it's to handle the case when more than one asset is selected in the asset tree.

Of another way should be to create an OrMultipleAssetsViewer component only dedicated to show the button to change parent for multiple selected asset in the tree.

Reply:
The one property `assetIds` should be sufficient and a single item in the array indicates single asset selection

Reply:
I need to checkout the code and see what happens to the asset viewer during multiselect as not clear to me why the asset viewer needs to know anything.